code/_globalvars/lists/quirks.dm

```diff
@@ -18,9 +18,11 @@ GLOBAL_LIST_INIT(limb_choice, list(
 	"Right Leg" = /obj/item/bodypart/leg/right/robot/surplus,
 ))
 
-GLOBAL_LIST_INIT(limb_choice_transhuman, list(
+GLOBAL_LIST_INIT(part_choice_transhuman, list(
 	"Left Arm" = /obj/item/bodypart/arm/left/robot,
 	"Right Arm" = /obj/item/bodypart/arm/right/robot,
 	"Left Leg" = /obj/item/bodypart/leg/left/robot,
 	"Right Leg" = /obj/item/bodypart/leg/right/robot,
+	"Robotic Voice Box" = /obj/item/organ/internal/tongue/robot,
+	"Flashlights for Eyes" = /obj/item/organ/internal/eyes/robotic/flashlight,
```

Comment:
do note that organs do not count as much as robotic limbs when it comes to determining your mood.
This means you may spawn with a  negative moodlet off the bat because you selected these instead of augments.

Reply:
yep, I think that pushing the player to actually go and _get_ more augs will make transhumanist more interesting, and it serves as a different weakness compared to the robotic limbs that need to be repaired rather then healed. If you select an organ, you get  ![firefox_p6MSUL2UZC](https://github.com/tgstation/tgstation/assets/49223093/c1cf38f9-2575-40fa-9b63-836cde1cc3de) at roundstart, to communicate this to the player.
but! obviously this is fair to be changed, if it was too annoying for anyone to actually play.